packages/flutter_tools/templates/app_shared/.gitignore.tmpl

```diff
@@ -1,4 +1,6 @@
+{{^withEmptyMain}}
```

Comment:
I don't feel very strongly, but I would prefer leaving this as is in both flows.

Reply:
Okay, I reverted it.